docs/source/upcoming_release_notes/1157-api_motion_pv_cull.rst

```diff
@@ -0,0 +1,43 @@
+1157 api_motion_pv_cull
+#######################
+
+API Changes
+-----------
+- Remove rarely-used TwinCAT state config PVs from `TwinCATStateConfigOne`
+  that are also being removed from the IOCs.
+  These are still available on the PLC itself.
+
+  - ``delta``
```

Comment:
`rst` rendering is always dicey for me, but looks like you got it right here - looks good 👍